web/components/Feed/Feed.tsx

```diff
@@ -232,6 +237,49 @@ const filterGroups: FilterGroup[] = [
   },
 ];
 
+function getPrefFilters(userData: IUser | null): SelectedFilters | null {
```

Comment:
Consider writing a JSDoc comment here to explain the function and what it returns